apps/ui/README.md

```diff
@@ -0,0 +1,50 @@
+# React + TypeScript + Vite
```

Comment:
Lets replace this template readme with one related to the application we are actually providing. 

For inspiration you can look at the OHCRN repo for package readme examples:
https://github.com/OHCRN/platform/blob/develop/apps/consent-api/README.md
https://github.com/OHCRN/platform/blob/develop/apps/consent-ui/README.md

Reply:
Thanks, I need permissions to view

Reply:
Removed auto-generated Vite content until this is resolved